From 79a45b3f0ca5630402936fb9a1398b6dc0fc7f7f Mon Sep 17 00:00:00 2001
From: Abhishek <apatgaonkar@habana.ai>
Date: Mon, 28 Jun 2021 12:40:17 +0300
Subject: [PATCH] [SW-47229] greco: legacy: set default pci id to addin card

PCI ID for greco ADDIN card(75W) is 0x0030 which should be set
as default because all cards available right now are ADDIN.
This patch does following change:
Default PCI ID in non-secure mode: 0x0030
Default PCI ID in secured mode: 0x0031

As there is no secured mode available in legacy flow so it
will always go for non-secured mode.

Change-Id: Ib808d3091c8427637095d7c80b4b9dd7654dfd85
---

diff --git a/arch/greco/common.c b/arch/greco/common.c
index a6d6213..599c0fa 100644
--- a/arch/greco/common.c
+++ b/arch/greco/common.c
@@ -265,7 +265,7 @@
 
 u32 preboot_device_id_get(void)
 {
-	return PCI_IDS_GRECO;
+	return PCI_IDS_GRECO_ADDIN;
 }
 
 void preboot_init_pcie(void)
